ibr/png-parser/create_ibr.py

```diff
@@ -1,4 +1,6 @@
 import sys
+import os
+from PIL import Image
```

Comment:
this isn't a standard library, also, do we need to include this python package in build/requirements.txt?